kaavapino/urls.py

```diff
@@ -1,11 +1,19 @@
 from django.conf import settings
 from django.conf.urls.static import static
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
+from rest_framework import routers
 
 admin.autodiscover()
 
-urlpatterns = [path("admin/", admin.site.urls)]
+
+router = routers.DefaultRouter()
+
+urlpatterns = [
+    path("admin/", admin.site.urls),
+    path("v1/", include(router.urls)),
```

Comment:
As talked about in slack, I don't see this as a nice way to adding routes, this would require all of the viewsets to be imported into this single urls file, which is not something that is usually done in Django. I find it much nicer if all urls.py files from separate apps are imported separately and added. So that would change this to look more like:

```
url(r'^v1/', include(projects.urls)),
url(r'^v1/', include(someotherapp.urls)),
```

Reply:
In other words, this line can be removed or changed to `url(r'^v1/', include(projects.urls)),` if such urls exists currently.

Reply:
Change of plans, lets go with https://stackoverflow.com/questions/31483282/django-rest-framework-combining-routers-from-different-apps#comment83565039_40904241